docs/reference/releases/rev156_126.md

```diff
@@ -0,0 +1,60 @@
+# Revision 156/126
+<sub>Wednesday, February 28, 2024</sub>
+
+Dear community,
+
+We are extremely thrilled and excited to share that Charmed Kafka and Charmed ZooKeeper have now been released as GA. You can find them in [charmhub.io](https://charmhub.io/) under the `3/stable` track.
```

Comment:
```suggestion
"We are thrilled to announce that [Charmed Kafka](https://charmhub.io/kafka) and [Charmed ZooKeeper](https://charmhub.io/zookeeper) have reached General Availability (GA). You can find them on [charmhub.io](https://charmhub.io/) under the `3/stable` track."
```

Reply:
I just did copy and paste of the annoucements we did in the public channel. I'll keep the phrasing as it was the original one.